README.rst

```diff
@@ -0,0 +1,152 @@
+Elm Docstyle
+============
+A tool that allows you to check the existence and quality of your `Elm <http://elm-lang.org/>`_ code comments.
+
+Usage
+=====
+
+Prerequisites
+
+
+The following binaries should be available on the path:
+
+
+.. code-block:: bash
+
+    node >=6
+    elm  0.18.x
+
```

Comment:
The usage is actually missing in this section -- you probably forgot to include an example of command-line invokation?

Reply:
It's actually below, my bad